docs/reference/UDS Core/prerequisites.md

```diff
@@ -136,3 +140,180 @@ Metrics server is provided as an optional component in UDS Core and can be enabl
+          - path: read.extraArgs
+            value:
+            - "-config.expand-env=true"
+          - path: read.extraEnv
+            value:
+            - name: LOKI_ACCESS_KEY_ID
+              valueFrom:
+                secretKeyRef:
+                  name: minio-loki
+                  key: accessKey
+            - name: LOKI_SECRET_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: minio-loki
+                  key: secretKey
+```
+
+#### Velero
+
+The Velero deployment is (by default) backed by an object storage provider for backup retention.  For cloud environments you can wire this into the environment's storage provider with the following overrides:
+
+```yaml
+velero:
+  values:
+    - path: credentials.secretContents.cloud
+      value: |
+        [default]
+        aws_access_key_id=<s3-access-key>
+        aws_secret_access_key=<s3-secret-key>
+    - path: "configuration.backupStorageLocation"
+      value:
+        - name: default
+          provider: aws
+          bucket: "<bucket-name>"
+          config:
+            region: "<s3-region>"
+            s3ForcePathStyle: true
+            s3Url: "<s3-endpoint>"
+          credential:
+            name: "velero-bucket-credentials"
+            key: "cloud"
+```
+
+You can also use the [Minio Operator UDS Package](https://github.com/defenseunicorns/uds-package-minio-operator) to back Velero with the following overrides:
+
+```yaml
+- name: minio-operator
+  ...
+  overrides:
+    minio-operator:
+      uds-minio-config:
+        values:
+          - path: apps
+            value:
+              - name: velero
+                namespace: velero
+                remoteSelector:
+                  app.kubernetes.io/name: velero
+                bucketNames:
+                  - uds-velero
+                copyPassword:
+                  enabled: true
+                  secretIDKey: AWS_ACCESS_KEY_ID
+                  secretPasswordKey: AWS_SECRET_ACCESS_KEY
+
+- name: core-backup-restore
+  ...
+  overrides:
+    velero:
+      velero:
+        values:
+          - path: "credentials"
+            value:
+              useSecret: true
+              existingSecret: "minio-velero"
+              extraEnvVars:
+                AWS_ACCESS_KEY_ID: dummy
+                AWS_SECRET_ACCESS_KEY: dummy
+          - path: "configuration.backupStorageLocation"
+            value:
+              - name: default
+                provider: aws
+                bucket: "uds-velero"
+                config:
+                  region: ""
+                  s3ForcePathStyle: true
+                  s3Url: "http://uds-minio-hl.minio.svc.cluster.local:9000/"
+```
+
+:::caution
+If you are using the in-cluster Minio Operator UDS Package for backups you must ensure that the volumes that back that storage are themselves backed up!  Cluster or deployment issues may result in a loss of Minio as well as the application you intend to back up.
+:::
```

Comment:
nit, but I wonder if there's a better place to put this caution note given that it applies to both Loki and Velero (and is currently "nested" under the Velero section). We could just copy/paste it to Loki but that doesn't feel right 😅

Reply:
Also, we we have any other recommendations for backups? For example, in the [UDS docs](https://uds.defenseunicorns.com/reference/uds-core/prerequisites/#default-storage-class), we wrote:

> For instance, EBS volumes have a minimum size of 1Gi, which could lead to unexpected behavior, especially during Velero’s CSI backup and restore process. These constraints may also necessitate adjustments to default PVC sizes, such as Keycloak’s PVCs, which default to 512Mi in devMode.

Perhaps we could align these bits of the documentation somehow?

Reply:
added a common caution under that